components/idp-mgt/org.wso2.carbon.idp.mgt.ui/src/main/resources/web/idpmgt/idp-mgt-edit.jsp

```diff
@@ -3774,7 +3777,7 @@ function doValidation() {
             </td>
             <td>
                 <div class="sectionCheckbox">
-                    <input id="sloEnabled" name="sloEnabled" type="checkbox" <%=sloEnabledChecked%>/>
+                    <input id="sloEnabled" name="sloEnabled" type="checkbox" <%=Encode.forHtmlAttribute(sloEnabledChecked)%>/>
```

Comment:
The variable value is hardcoded. Further this is not an attribute. The variable fully represents the attribute key and value here. So no need to encode.